lib/broadside/configuration/deploy.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
```

Comment:
i was actually hoping to add a lot of specs with the `stub_responses` part of the `Aws` gem but i wanted to get some buy in that this approach basically makes sense before doing that (hence the `WIP` in the title)